bindings/rascal/representations/spherical_expansion.py

```diff
@@ -87,8 +87,31 @@ class documentation
                 unit='A'
             ),
         )
+
+        if 'type' in optimization_args:
+            if optimization_args['type'] == 'Spline':
+                if 'accuracy' in optimization_args:
+                    accuracy=optimization_args['accuracy']
+                else:
+                    accuracy=1e-8
```

Comment:
Absolute for mean error, but can be easily changed by using a different ErrorMethods as template parameter (look at Interpolator_t spherical expansion file). numpy/scipy actually calculates relative and absolute and stops as soon as one confirms the accuracy. This would be handy, because for Hyp1f1 the relative error works better because the values are very small while for the RadialContribution it is the opposite.